app/controllers/assignments_controller.rb

```diff
@@ -31,30 +31,30 @@ def create
     end
   end
 
-	def edit
-		#render the edit page. :assignmentExists should populate @assignment
+  def edit
+    # render the edit page. :assignmentExists should populate @assignment
     @assignment = Assignment.find(params[:assignment_id])
     @course = @assignment.course
-	end
+  end
 
-	def update
-		@update = Assignment.new(assignment_params)
-		if @update.valid?
-			Assignment.update(@assignment.id, assignment_params)
+  def update
+    @update = Assignment.new(assignment_params)
+    if @update.valid?
+      Assignment.update(@assignment.id, assignment_params)
       flash[:message] = "You have edited this assignment"
       redirect_to assignment_show_path(@assignment) and return
     else
-    	render 'edit' and return
-		end
-	end
+      render 'edit' and return
```

Comment:
Use `&&` instead of `and`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.